src/main/java/com/mycompany/myapp/domain/TaskComment.java

```diff
@@ -34,17 +35,18 @@
     private Task task;
 
     // jhipster-needle-entity-add-field - JHipster will add fields here
+
```

Comment:
Remove extra line for 1st field

Reply:
The generator code doesn't add this empty line, I guess it might be prettier.
```

    // jhipster-needle-entity-add-field - JHipster will add fields here
<%_ if (!embedded && primaryKey.length > 1) { _%>
    public <% if (databaseType === 'sql') { %><%= primaryKeyType %><% } %><% if (databaseType === 'mongodb' || databaseType === 'couchbase' || databaseType === 'neo4j') { %>String<% } %><% if (databaseType === 'cassandra') { %>UUID<% } %> getId() {
        return id;
    }
```

Reply:
this is inconsistent with 
// jhipster-needle-entity-add-getters-setters - JHipster will add getters and setters here
we keep an empty line before and after getter and setter we should keep an empty line here.

Reply:
if this is ok with you please mark it as resolved, I resolved all the others are they are fixed now